concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -1091,21 +1096,25 @@ instance (Monad m) => MHashableTo m Hash.Hash (PersistentAccount 'AccountV3)
 instance (Monad m) => MHashableTo m Hash.Hash (PersistentAccount 'AccountV4)
 instance (Monad m) => MHashableTo m Hash.Hash (PersistentAccount 'AccountV5)
 
-instance (MonadBlobStore m, IsAccountVersion av) => BlobStorable m (PersistentAccount av) where
+instance (MonadBlobStore m, IsAccountVersion av, SingI (SupportsPLT av)) => BlobStorable m (PersistentAccount av) where
```

Comment:
I didn't manage to derive the `BlobStorable` instance for `Conditionally` as the default implementation derived from its `Serialize` instance. This isn't needed, if we get the instance otherwise.